tools/PI/DevHome.PI/Controls/AddToolControl.xaml.cs

```diff
@@ -84,7 +84,12 @@ private void HandleBrowseButton()
     {
         // WinUI3's OpenFileDialog does not work if we're running elevated. So we have to use the old Win32 API.
         var fileName = string.Empty;
-        var filter = "Executables (*.exe)\0*.exe\0Batch Files (*.bat)\0*.bat\0\0";
+        var filter = "Executables (*.exe)\0*.exe\0" +
+             "Batch Files (*.bat)\0*.bat\0" +
+             "Command Files (*.cmd)\0*.cmd\0" +
+             "Management Console Files (*.msc)\0*.msc\0" +
+             "PowerShell Scripts (*.ps1)\0*.ps1\0" +
```

Comment:
> "PowerShell Scripts (*.ps1)\0*.ps1\0" + [](http://example.com/codeflow?start=12&length=40)

should we add support for .py here as well?

Reply:
Yes that's reasonable. We're kinda covered with the All Files *.* - but per PR notes above, I've only tested the ones we explicitly list. I'm interested to get customer feedback on what's the optimal set to include. As above, deferred to a further round later.